sys-libs/hwloc/patches/hwloc-2.10.0.patchset

```diff
@@ -0,0 +1,22 @@
+From 073ee587e085d07e00ca6bef497d5e769880f579 Mon Sep 17 00:00:00 2001
+From: Damillora <duniapengetahuan.12345@gmail.com>
+Date: Sun, 3 Dec 2017 11:19:42 +0000
+Subject: Fix asm error
```

Comment:
Re-instated this from prior version, if not really needed it can still be removed.